projects/components/CHANGELOG.MD

```diff
@@ -8,6 +8,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 ### Added
 [internal]: Update `ng-live-docs` to 0.0.8 to enable `@vmw/ng-live-docs:add-example` schematic.
+### Changed
+1. BREAKING: Rewrote `VcdDatagridWidget` and `ClrDatagridWidget` with new Widget Object.
```

Comment:
Can you switch this to be the notation that Juan uses on line 10 with the brackets: []?

Reply:
I don't remember doing line 10. We lose the list if we do that.